extensions/src/main/java/org/pentaho/platform/web/http/api/resources/services/UserRoleListService.java

```diff
@@ -112,6 +114,19 @@ public RoleListWrapper getSystemRoles() {
     return new RoleListWrapper( systemRoles );
   }
 
+  public RoleListWrapper getAllRoles( boolean excludeAnonymous ) {
+    if ( !excludeAnonymous ) {
+      return getAllRoles();
+    }
```

Comment:
Please, do this the other way around, by changing `getAllRoles()` to call this one. Will allow you not have to create a new role list wrapper.

Reply:
Addressed